sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/AzureFileStorageImplBuilder.java

```diff
@@ -6,16 +6,20 @@
 
 import com.azure.core.annotation.ServiceClientBuilder;
 import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpHeaders;
```

Comment:
What are the changes in this file for?  They seems unrelated to premium file bandwidth limits.

Reply:
It's a generated file, so I think these must be changes that came out of whatever updates were made to the generator.